src/Chirp.Core/Helpers/DBFacade.cs

```diff
@@ -10,12 +10,17 @@ namespace Chirp.Core.Helpers
 {
     public class DBFacade : IDisposable
     {
-        private const string _sqlDBFilePath = "../Chirp.Core/Assets/chirp.db";
```

Comment:
This needs to be and environment variable as stated [in the lecture notes](https://github.com/itu-bdsa/lecture_notes/blob/main/sessions/session_05/README_PROJECT.md)

> Your program shall receive the location from the SQLite database file from an environment variable called CHIRPDBPATH, i.e., via dependency injection. In case no such variable is defined the database file shall be located under a [user's temporary directory](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.gettemppath?view=net-7.0&tabs=linux) with the file name chirp.db. That is, calling your program directly, e.g., with dotnet run will store the database file under <user's tempdir>, whereas calling it with, e.g., CHIRPDBPATH=./mychirp.db dotnet run will store the file in the current directory under the file name mychirp.db.

Reply:
Also this path does not work for azure. I don't really know what would though.

Reply:
Yes, that is still needed. Which is what I stated in my pull request description haha